### PR TITLE
Feature(docker-examples): Developers should be aware of docker examples

### PR DESCRIPTION
**Pull request by michaelachrisco**
## Changes
1. Added docker examples from boilerplate projects

## Purpose
Add more detail to our docker docs.

## Approach
To provide real docker examples used within our projects.

References #174

### CODE REVIEW

File: best-practices/development-tools/docker/README.md [hunk 1]
```diff
@@ -39,6 +39,10 @@ Docker Compose File Documentation:
   - [Nodejs](./nodejs)
   - [Django](./django)
 
+## Examples from Internal Projects:
+  - [node-boilerplate](https://github.com/Shift3/boilerplate-client-angular#docker)
```
Comment (Basim-Elayan) [inline best-practices/development-tools/docker/README.md:43]: Not a big issue, I incline to unify this link with the bottom one by using https://github.com/Shift3/boilerplate-client-angular/blob/main/Dockerfile 
Unless it's in purpose.

File: .circleci/config.yml [hunk 2]
```diff
@@ -7,4 +7,4 @@ jobs:
...
+      - run: bundle exec awesome_bot --allow-redirect --allow-dupe -w awesome_bot --allow-redirect --allow-dupe -w https://github.com/Shift3/ifg/compare/v0.4.0...v0.5.0,https://github.com/Shift3/sinclair-scanner-application,https://github.com/Shift3/sinclair-scanner-application/pull/137,https://github.com/Shift3/FUSDMonitoringAppXamarin,https://github.com/Shift3/rn-by-example,https://github.com/Shift3/fusd-front-desk-kiosk,regexr,https://github.com/Shift3/bw-project-status,https://github.com/Shift3/Aerostar,https://github.com/Shift3/qa-team/projects,https://github.com/Shift3/sundial-mobile-app,https://github.com/Shift3/boilerplate-client-angular#docker,https://github.com/Shift3/boilerplate-server-node/blob/develop/docker-compose.yml,127.0.0.1 `find . -name "*.md"`
```
Comment (Basim-Elayan) [inline .circleci/config.yml:10]: It would be helpful to explain the concept of this command.

**Review by Basim-Elayan — APPROVED** [review]
LGTM., nice addition

File: best-practices/development-tools/docker/README.md (hunk 1 above)
Comment (michaelachrisco) [reply best-practices/development-tools/docker/README.md:43]: hmm good point, ill make the change.

File: .circleci/config.yml (hunk 2 above)
Comment (michaelachrisco) [reply .circleci/config.yml:10]: Awesome_bot checks all links within READMEs recursively to make sure they are still valid. The edit comes from a series of Bitwise/Shift3 specific links that cannot be checked as they are private repos. 

Im thinking of removing awesome_bot entirely from individual PRs like this one and changing it to a semi-weekly cron job kind of setup within circleci but its still in the works. For now, this has proven to work on keeping links up to date.

File: best-practices/development-tools/docker/README.md [hunk 3]
```diff
@@ -39,6 +39,10 @@ Docker Compose File Documentation:
   - [Nodejs](./nodejs)
   - [Django](./django)
 
+## Examples from Internal Projects:
+  - [node-boilerplate](https://github.com/Shift3/boilerplate-client-angular/blob/main/docker-compose.yml)
+  - [angular-boilerplate](https://github.com/Shift3/boilerplate-server-node/blob/develop/docker-compose.yml)
```
Comment (mwallert) [inline best-practices/development-tools/docker/README.md:44]: The labels for these links are reverse.

Comment (michaelachrisco) [reply best-practices/development-tools/docker/README.md]: Fixed via 9a8cf990f467cd7acadda6e361af0f56484cd422

I feel silly, thanks for looking that over.

Comment (mwallert) [reply best-practices/development-tools/docker/README.md]: @michaelachrisco Don't worry about it, could have happened to anyone!

File: best-practices/development-tools/docker/README.md (hunk 1 above)
Comment (michaelachrisco) [reply best-practices/development-tools/docker/README.md]: fixed via: 8ebeb097922d8c4531b2561a1d4b257683a4ed90

**Review by mwallert — APPROVED** [review]
LGTM

File: .circleci/config.yml (hunk 2 above)
Comment (Basim-Elayan) [reply .circleci/config.yml]: Oh super cool. good to know about it. Thanks for clarifying that for me.

**Pull request merged**